gulp/tasks/test.js

```diff
@@ -93,6 +94,36 @@ function _getProtractorParams() {
   return params;
 }
 
+gulp.task( 'test:acceptance:perf', sitespeedio( {
+  url: 'http://localhost:8000/',
```

Comment:
Sure. I pulled in those variables in 5b36292, but let me know if you see a better way to do that.